packages/shell-chrome/src/backend.js

```diff
@@ -189,6 +190,27 @@ function discoverComponents() {
     )
 }
 
+const _dispatchEvent = EventTarget.prototype.dispatchEvent
```

Comment:
Btw, this will cause a crash on backend.js re-injection (eg. Close & reopen devtools)

I've been tempted to refactor the bulk of the stateful bits of this file (observer, id, hoverElement) into a class which should.solve that without the whole `window.__alpineDevtool` shenanigans

Reply:
Okay, thanks for the notice. I wonder whether this task is better off waiting for a refactor / rewrite of communication, etc...